### PR TITLE
[ONNX] Support stripped ONNX

### PR DESCRIPTION
**Pull request by disktnk**
When running `strip_large_tensor` with ONNX which has already stripped, occur an error

```
FileNotFoundError: [Errno 2] No such file or directory: '/path/{"type": "stripped", "average": 0.0002447886217851192, "variance": 0.004106185864657164}'
```

Because stripped meta information is written as "external data location" and `onnx.load` try to loading the external location path https://github.com/onnx/onnx/blob/685898bc6f3fd6eb16ba129ea990f96330537e38/onnx/external_data_helper.py#L47

This PR fixed the error, by skipped stripped tensor when (re)stripping.

### CODE REVIEW

**Comment by kmaehashi** [comment]
I'm reviewing but would be great if you could double-check @take-cheeze 😄

File: pytorch_pfn_extras/onnx/strip_large_tensor.py [hunk 1]
```diff
@@ -16,6 +16,22 @@ def is_large_tensor(tensor, threshold):
     return size > threshold
 
 
+def _is_stripped(tensor):
+    if len(tensor.external_data) == 0:
+        return False
+    external_value = None
+    for external_data in tensor.external_data:
+        if external_data.key == 'location':
+            external_value = external_data.value
```
Comment (kmaehashi) [inline pytorch_pfn_extras/onnx/strip_large_tensor.py:25]: `location` won't appear twice? (correct me if I'm wrong)
```suggestion
            external_value = external_data.value
            break
```

File: pytorch_pfn_extras/onnx/strip_large_tensor.py [hunk 2]
```diff
@@ -16,6 +16,22 @@ def is_large_tensor(tensor, threshold):
...
+    if not external_value:
+        return False
+    try:
+        external_value_dict = json.loads(external_value)
+        return external_value_dict.get('type', '') == 'stripped'
+    except Exception:
+        return False
```
Comment (kmaehashi) [inline pytorch_pfn_extras/onnx/strip_large_tensor.py:32]: Let's use the narrower exception.

```suggestion
    except ValueError:
        # Invalid JSON, indicating `external_value` contains a file path
        return False
```

**Review by kmaehashi — CHANGES_REQUESTED** [review]
LGTM except minor comments!

**Comment by kmaehashi** [comment]
For later reference: https://github.com/onnx/onnx/blob/master/docs/ExternalData.md

**Review by take-cheeze — APPROVED** [review]

File: pytorch_pfn_extras/onnx/strip_large_tensor.py (hunk 2 above)
Comment (disktnk) [reply pytorch_pfn_extras/onnx/strip_large_tensor.py]: fixed

File: pytorch_pfn_extras/onnx/strip_large_tensor.py (hunk 1 above)
Comment (disktnk) [reply pytorch_pfn_extras/onnx/strip_large_tensor.py]: Before code could not consider multiple external data, so fixed to handle them.

**Comment by take-cheeze** [comment]
Sorry I've miss-approved https://github.com/pfnet/pytorch-pfn-extras/pull/208

File: pytorch_pfn_extras/onnx/strip_large_tensor.py [hunk 3]
```diff
@@ -16,6 +16,22 @@ def is_large_tensor(tensor, threshold):
     return size > threshold
 
 
+def _is_stripped(tensor):
+    if len(tensor.external_data) == 0:
+        return False
```
Comment (kmaehashi) [inline pytorch_pfn_extras/onnx/strip_large_tensor.py:21]: I think this condition can now be removed.
```suggestion
```

**Review by kmaehashi — COMMENTED** [review]
LGTM except for minor comment.

File: pytorch_pfn_extras/onnx/strip_large_tensor.py (hunk 3 above)
Comment (disktnk) [reply pytorch_pfn_extras/onnx/strip_large_tensor.py]: fixed

**Review by take-cheeze — APPROVED** [review]
LGTM 👍

**Review by kmaehashi — APPROVED** [review]

**Pull request merged**